examples/jquery.html

```diff
@@ -1,10 +1,14 @@
+<!doctype html>
 <html>
   <head>
     <meta content='text/html; charset=utf-8' http-equiv='Content-Type'>
     <meta name="viewport" content="width=device-width,initial-scale=1">
+    <!-- Make example work in IE7 -->
+    <script src="//cdn.jsdelivr.net/console-log/2.2/console.min.js"></script>
```

Comment:
There's a console.log inside the code, IE7 has no console.

That's why "<!-- Make example work in IE7 -->"